app/models/shipit/task.rb

```diff
@@ -312,6 +312,16 @@ def permalink
       Shipit::Engine.routes.url_helpers.stack_task_url(stack, self)
     end
 
+    def commit_to_revert_to
```

Comment:
`commit_to_rollback_to`? `revert` implies git things.

Reply:
We already use `trigger_revert` in deploy.rb so I was re-using the same wording. But `rollback_to` does reads a bit better.

Reply:
@JackTLi What do you think makes more sense here?

Reply:
I've gone with `_rollback` as it's a bit more explicit and `Task` has no notion of 'revert' inside it anywhere.

Reply:
👍 to rollback over revert